lightwood/analysis/analyze.py

```diff
@@ -75,7 +75,8 @@ def model_analyzer(
         'tss': tss,
         'ts_analysis': ts_analysis,
         'accuracy_functions': accuracy_functions,
-        'has_pretrained_text_enc': has_pretrained_text_enc
+        'has_pretrained_text_enc': has_pretrained_text_enc,
+        'library_version': lightwood.__version__
```

Comment:
Tests are failing (you can check [here](https://github.com/mindsdb/lightwood/actions/runs/3206717079/jobs/5240808895) for more information). Try to `from lightwood import __version__` instead to avoid this particular exception.